README.md

```diff
@@ -1,2 +1,139 @@
 # conda-tracker
-Link git repositories and patches into an aggregate repository
+
+conda-tracker is a tool that links git repositories and patches into an aggregate 
+repository. By grouping repositories and patches into a single repository, tasks
+such as updating conda recipes prior to submitting to conda-concourse-ci become
+much easier.
+
+## Installation
+
+To install conda-tracker with conda:
+
+    $  conda install conda-tracker
+
+To install conda-tracker from source:
+
+    $  git clone https://github.com/conda/conda-tracker.git
+    $  cd conda-tracker
+    $  python setup.py install
+
+## Usage
+
+### Command Line Application
+
+conda-tracker provides a CLI wrapped around a library. The CLI may be invoked
+as follows:
+
+    usage: conda-tracker [OPTIONS] COMMAND [ARGS]...
+    
+    commands:
+      add           Add a repository to the aggregate repository
+      update        Update a linked repository to the master branch
```

Comment:
Probably should be better stated as "update submodule to a different revision on currently tracked branch", since submodules may be tracking branches other than master.